digitalocean/droplet/sweep.go

```diff
@@ -33,15 +33,18 @@ func sweepDroplets(region string) error {
 	}
 	log.Printf("[DEBUG] Found %d droplets to sweep", len(droplets))
 
+	var swept int
 	for _, d := range droplets {
-		if strings.HasPrefix(d.Name, "foo-") || strings.HasPrefix(d.Name, "bar-") || strings.HasPrefix(d.Name, "baz-") || strings.HasPrefix(d.Name, "tf-acc-test-") || strings.HasPrefix(d.Name, "foobar-") {
+		if strings.HasPrefix(d.Name, sweep.TestNamePrefix) {
```

Comment:
This is the main benefit of these change. Instead of needing to check for multiple different patterns before deleting, we just need to look for one. It simplifies things and should make the sweeper a safer to run.